ext/fs/interface.rs

```diff
@@ -105,7 +105,7 @@ impl From<io::Error> for FsError {
 pub type FsResult<T> = Result<T, FsError>;
 
 #[async_trait::async_trait(?Send)]
-pub trait File {
+pub trait File: deno_core::Resource {
```

Comment:
I don't think this bound is needed as long as the bound exists on the extension.

I don't want interface.rs to depend on core, because I want to move it to a separate crate that doesn't depend on core.

In the same spirit, the into_resource should be a different trait.

Reply:
Can you expand on what you mean? I don't see a way to make this work.

Reply:
Mh, I see now why this doesn't work. Because you never create a concrete type for `File`.

Have you considered implementing `FileSystem` on the `Box<T>` rather than on `T`? Maybe all of this is unneeded.